workshops/migrations/0012_auto_20150612_0807.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+TRANSLATE_NAMES = {
+    'Git': ['swc/git-novice'],
+    'Make': [],
```

Comment:
Not in a usable state - Mike Jackson has just offered us a rework that's 
much more usable, but it isn't ready for prime time yet.